django_slack_oauth/models.py

```diff
@@ -18,8 +15,23 @@ class SlackUser(models.Model):
     def is_slacked(self):
         return self.access_token
 
-    def __unicode__(self):
-        return unicode(self.slacker)
+    def __str__(self):
+        return self.slacker.__str__()
 
     class Meta:
         db_table = 'slack_user'
+
+
+class SlackOAuthRequest(models.Model):
```

Comment:
@izdi, I think we should deprecate `SlackUser` because depending on the scopes that you define, the request may not be creating a user at all. So it seems more general to call it a `SlackOAuthRequest`. 

Also, don't you think it's better to let Django namespace the table instead of forcing a name? This way, if the user already has a `slack_user` table—which is probably likely they are building a Slack app—we don't cause conflicts.